core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -135,6 +150,18 @@ object Controller {
 
     def optionalProperties = EntitlementProvider.optionalProperties
 
+    private def info(config: WhiskConfig, runtimes: Runtimes) = JsObject(
+        "description" -> "OpenWhisk".toJson,
+        "support" -> "https://github.com/openwhisk/openwhisk/issues".toJson,
+        "build" -> config(WhiskConfig.whiskVersionDate).toJson,
+        "build_tag" -> config(WhiskConfig.whiskVersionBuildno).toJson,
+        "api_paths" -> List("/api/v1").toJson,
```

Comment:
Thinking about api/v2. Maybe the version here should be passed in via a parameter.